.vscode/settings.json

```diff
@@ -0,0 +1,46 @@
+{
+    // Python settings
+    "python.analysis.typeEvaluation.strictDictionaryInference": true,
+    "python.analysis.typeEvaluation.strictListInference": true,
+    "python.analysis.typeEvaluation.strictSetInference": true,
+    "python.missingPackage.severity": "Error",
+
+    "debug.console.closeOnEnd": true,
+    "debugpy.debugJustMyCode": false,
+    "diffEditor.ignoreTrimWhitespace": false,
+    // add me when pixi "python.defaultInterpreterPath": "${workspaceFolder}/.venv/bin/python",
+    // add me when pixi "python.testing.pytestPath": "${workspaceFolder}/.venv/bin/pytest",
+    "python.analysis.autoSearchPaths": true,
```

Comment:
https://code.visualstudio.com/docs/languages/json#_json-with-comments

apparently vscode has some special type of json that allows comments, so we can leave it since its in the .vscode folder anyway?

Reply:
aaah okay that I didn't know, yeah then we can leave it